packages/layout/src/util/utils.ts

```diff
@@ -1,4 +1,7 @@
 // TODO: Actually I don't really like the union here (we have to convert the string value back to the enum value, if we want to use Enum somewhere)
+
+import { Position } from "@dataspecer/core-v2/visual-model";
+
 // TODO: So maybe just do the classic ... type Direction = "UP" | "RIGHT" | "DOWN" | "LEFT";
 export enum DIRECTION {
```

Comment:
I would name this "Direction" not "DIRECTION" the second version is for constatns not enums.